runtime/backtrace.c

```diff
@@ -42,10 +42,9 @@ CAMLprim value caml_record_backtrace(value vflag)
     Caml_state->backtrace_active = flag;
     Caml_state->backtrace_pos = 0;
     if (flag) {
-      Caml_state->backtrace_last_exn = caml_create_root(Val_unit);
+      Caml_state->backtrace_last_exn = Val_unit;
```

Comment:
I'm not convinced that `caml_register_generational_global_root` is called on `Caml_state->backtrace_last_exn` because I don't think `caml_init_backtrace` is called anywhere.